distro/exporter.go

```diff
@@ -298,3 +305,27 @@ func noneEnvVarSet(envs ...string) bool {
 func notNone(s string) bool {
 	return s != "" && s != "none"
 }
+
+func getOTLPProtocol(l logr.Logger, signalKey string) string {
```

Comment:
in Go you try to avoid `get` in the function/method names. Related to: https://go.dev/doc/effective_go#Getters

```suggestion
func otlpProtocol(l logr.Logger, signalKey string) string {
```